tools/build.py

```diff
@@ -985,8 +985,23 @@ def copytree(src, dst, symlinks=False, ignore=None):
                         os.makedirs(os.path.dirname(d))
                     print(shutil.copy2(s, d))
 
+
+
         rvpkg_staging = os.path.join(tempfile.mkdtemp(), 'rvpkg')
 
+        # BUILD DEPENDENCIES ZIP
+        subprocess.check_call(
```

Comment:
Try if the `append_dependencies` function will also work for this.

Reply:
I checked, but other parts of the code make use of a similar pip install solution, for now I'd keep this (just to release), once we review the whole plugin, I'm happy to look into other changes.